node/core/approval-voting/src/persisted_entries.rs

```diff
@@ -353,12 +386,21 @@ pub struct BlockEntry {
 	// block. The block can be considered approved if the bitfield has all bits set to `true`.
 	pub approved_bitfield: Bitfield,
 	pub children: Vec<Hash>,
+	// A list of candidates that has been approved, but we didn't not sign and
+	// advertise the vote yet.
+	pub candidates_pending_signature: BTreeMap<CandidateIndex, CandidateSigningContext>,
```

Comment:
IMO mixed public/private fields in a struct is a code smell.
I know this got introduced recently, but smells multiply.

Reply:
Agree this didn't respect the local style, I'll address it.